qiskit/chemistry/drivers/pyscfd/integrals.py

```diff
@@ -104,36 +108,40 @@ def _check_molecule_format(val):
     return val
 
 
-def _calculate_integrals(mol, hf_method='rhf', conv_tol=1e-9, max_cycle=50, init_guess='minao'):
-    """Function to calculate the one and two electron terms. Perform a Hartree-Fock calculation in
-        the given basis.
-    Args:
-        mol (gto.Mole) : A PySCF gto.Mole object.
-        hf_method (str): rhf, uhf, rohf
-        conv_tol (float): Convergence tolerance
-        max_cycle (int): Max convergence cycles
-        init_guess (str): Initial guess for SCF
-    Returns:
-        QMolecule: QMolecule populated with driver integrals etc
-    Raises:
-        QiskitChemistryError: Invalid hf method type
-    """
-    enuke = gto.mole.energy_nuc(mol)
+def _create_mol_and_run_scf(atom, unit, charge, spin, basis, hf_method='rhf', conv_tol=1e-9,
+                            max_cycle=50, init_guess='minao', max_memory=None):
+    """Create molecule and run SCF calculation"""
```

Comment:
This function is called from the driver, type hints and a docstring would be nice to have.